src/url-params.ts

```diff
@@ -15,6 +15,7 @@ const deviceIdSeparator = /\s*,\s*/;
  */
 export class AnalyticsUrlParams {
 	private deviceIds: Set<string> = new Set();
+	private passedDeviceId: string | null = '';
```

Comment:
Can you please not set this to empty string by default Ezequiel?
```suggestion
	private passedDeviceId: string | null;
```

Just to prevent any potential issues down the line.